community/font-awesome/sources

```diff
@@ -0,0 +1 @@
+https://github.com/FortAwesome/Font-Awesome/archive/5.12.1.tar.gz
```

Comment:
```
FortAwesome
```

Typo? Did you not build this package before submitting?

Reply:
I build the package before submitting. Did you check the link before commenting? Now seriously, the source link is correct.

Reply:
Well, that's unexpected. :woman_shrugging: